src/app/builder/builder.component.html

```diff
@@ -107,9 +111,16 @@
         </fieldset>
 
         <fieldset>
-          <img [src]="props.imageUrl || defaults.imageUrl" id="artwork-preview">
-          <label>Artwork URL</label>
-          <input type="text" [(ngModel)]="props.imageUrl" id="imageUrl" [placeholder]="defaults.imageUrl" name="imageUrl">
+          <img [src]="props.epImageUrl || defaults.epImageUrl" id="artwork-preview">
+          <label>Episode Artwork URL</label>
+          <p><small>This artwork will be featured prominently in the foreground of the player.</small></p>
+          <input type="text" [(ngModel)]="props.epImageUrl" id="epImageUrl" [placeholder]="defaults.epImageUrl" name="epImageUrl">
```

Comment:
I had put it as placeholder rather than default originally because back when I was working on the original GH ticket to show the default player values pulled from the RSS, the consensus was to have them visible but unselectable (I don't totally remember why tbh -- possibly to discourage users from accidentally typoing and then not knowing why their player doesn't work?)